advocacy_docs/supported-open-source/pgbackrest/08-multiple-repositories.mdx

```diff
@@ -0,0 +1,590 @@
+---
+title: 'Multiple Repositories'
+description: "Deeper look at the multiple repositories support feature"
+---
+
+The multiple repositories support was added in the [v2.33](https://github.com/pgbackrest/pgbackrest/releases/tag/release%2F2.33) release. One example where multiple repositories could be used, would be for situations where a local repository would have minimal retention (for fast restores), and a remote repository would have a longer retention (for redundancy and access across the enterprise).
+
+A new [`--repo`](https://pgbackrest.org/configuration.html#section-repository/option-repo) option has been added to set the repository for a command to operate on. All `repo*-` options are indexed to allow for configuring multiple repositories.
+
+In this section, we'll setup a **demo** cluster (using **EDB Postgres Advanced Server** 13 on CentOS 7) to demonstrate how each pgBackRest command reacts to this new feature.
+
+
+### Configuration
+
+For the purpose of this demonstration, we'll setup two local repositories on our database host. This is **not recommended** for production purpose.
+
+```ini
+[global]
+# repository details
+repo1-path=/var/lib/edb/as13/backups/repo1
+repo1-retention-full=1
+repo2-path=/var/lib/edb/as13/backups/repo2
+repo2-retention-full=1
+
+# general options
+process-max=2
+log-level-console=info
+log-level-file=debug
+start-fast=y
+delta=y
+
+[demo]
+pg1-path=/var/lib/edb/as13/data
+pg1-user=enterprisedb
+pg1-port=5444
+```
+
+For safety reasons, the `--repo` option can't be defined in the configuration file or a warning will be triggered: `WARN: configuration file contains command-line only option 'repo'`. If there is more than one repository configured and the `--repo` option is not specified for a command, the repositories will be acted upon in highest priority order (e.g. `repo1` then `repo2`).
```

Comment:
Require more clarity

Reply:
@ManjushaVaidya: How? I mean if I wrote it like that, it's because I couldn't imagine how to phrase it differently. I also want to emphases that this WARNing is expected and why it's there.

Basically, if the mutli repo feature is used (setting several repos in the config) but the --repo option isn't used in the command arguments, then the command will only act on the repository with the highest priority order (repo1, then repo2, then repo3,...)...

Reply:
... the repository with the highest priority (e.g. `repo1`) will be chosen by default.

Reply:
fixed in 8f97564